backend/handlers/handlers.go

```diff
@@ -1032,6 +1036,44 @@ func (handlers *Handlers) postExportAccountSummary(_ *http.Request) (interface{}
 	return path, nil
 }
 
+func (handlers *Handlers) getExchangesByRegion(r *http.Request) (interface{}, error) {
+	type errorResult struct {
+		Error string `json:"error"`
+	}
+	regions, err := exchanges.ListExchangesByRegion(handlers.backend.HTTPClient())
+	if err != nil {
+		handlers.log.Error(err)
+		return errorResult{Error: err.Error()}, nil
+	}
+	return regions, err
```

Comment:
`return regions, nil` is idiomatic, otherwise it's a bit confusing and looks like a real error is returned.

Reply:
Thanks, fixed